### PR TITLE
Simorgh Local Toggle for AMP ads

### LINKED ISSUE #7382: Add local Simorgh toggle for AMP ads
**Is your feature request related to a problem? Please describe.**
We want to enable canonical ads on live first, and then enable AMP ads once we have sorted all issues with AMP consent. In order to do this, we should create a new local toggle. We can can use this to ensure that we can release AMP ads independently of canonical ads.

**Describe the solution you'd like**
Add a new local toggle for AMP ads, and if this is off, then don't render ads on AMP pages (even if display ads are toggled on).

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by itimirichard**
Resolves #7382

**Overall change:**
_ Added a new toggle for enabling and disabling ads on AMP regardless of the the `ads` toggle value._

**Code changes:**

- _Added a new toggle `ampAds` to all the env config files._
- _AdContainer shoud return null when on and AMP page and ampAds is toggle off._
- _Added assertions for the case when `ampAds` is toggle off and updated snapshots accordingly._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/toggles/localConfig.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ export default {
   ads: {
     enabled: false,
   },
+  ampAds: {
+    enabled: false,
```
Comment (thekp) [inline src/app/lib/config/toggles/localConfig.js]: I think AMP Ads are worth keeping toggled on for LOCAL and maybe TEST environments so when people are developing amp-ads they don't have to update this value.

**Review by thekp — COMMENTED** [review]
LGTM overall just left a small comment that I think we should consider.

File: src/app/lib/config/toggles/localConfig.js (hunk 1 above)
Comment (tochwill) [reply src/app/lib/config/toggles/localConfig.js]: Yep, agreed with this - let's have them enabled on local and test, then this PR looks good to me!

Comment (itimirichard) [reply src/app/lib/config/toggles/localConfig.js]: Thanks. This has been done

File: src/app/containers/Ad/index.jsx [hunk 2]
```diff
@@ -12,8 +12,9 @@ const AdContainer = ({ slotType }) => {
   const { ads } = useContext(ServiceContext);
   const hasAds = pathOr(false, ['hasAds'], ads);
   const { enabled: adsEnabled } = useToggle('ads');
+  const { enabled: ampAdsEnabled } = useToggle('ampAds');
 
-  if (!hasAds || !adsEnabled) {
+  if (!hasAds || !adsEnabled || (isAmp && !ampAdsEnabled)) {
```
Comment (tochwill) [inline src/app/containers/Ad/index.jsx]: This isn't quite the logic we wan't - if `!adsEnabled`, then we don't want to render anything. If adsEnabled is true, then we need to check if amp ads are enabled before rendering AMP ads. Maybe we should just check the ampAdsEnabled toggle in the amp ad container?

Comment (itimirichard) [reply src/app/containers/Ad/index.jsx]: oops! Yeah, didn't see that!

**Review by thekp — APPROVED** [review]

**Review by hotinglok — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**